skipruntime-ts/ts/examples/package.json

```diff
@@ -1,5 +1,8 @@
 {
   "type": "module",
+  "scripts": {
+    "lint": "eslint --config ../../../eslint.config.js ."
```

Comment:
Note:
I intentionally did not enable eslint and very strict tsc rules on tests and examples because we probably won't expect it from our users.
I.e. I don't want to "cheat" when writing the examples.
Not sure it's very reasonable though

Reply:
Hmm, I was thinking that the example code should be a demonstration of what we want to encourage users to do, and so making TS as close to a reasonable language as possible there as well seems like the way to go. I don't know though. I just found myself duplicating the config and so deduped it.